bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToZATo2L2B_ggfusion_b-associatedproduction/example_cards/HToZATo2L2B_500p00_300p00_1p50_ggH_TuneCP5_13TeV_pythia8/HToZATo2L2B_500p00_300p00_1p50_ggH_TuneCP5_13TeV_pythia8_proc_card.dat

```diff
@@ -1,4 +1,4 @@
 import model 2HDMtII_NLO
 #gg fusion -loop induced
-generate p p > h2 > h3 Z [QCD] @0
+generate p p > h2 > h3 l+ l- $$ a [QCD] @0
```

Comment:
Why do you need $$ a ?

Reply:
`$$ a` to remove the diagrams that include `l+l-` coming from the photons decay which is not resonance in my search. We want to keep only the process that we are interested in ` pp > h2 > h3 Z and the Z decay to l+ l-`.

Reply:
But you do mll > 50. In your analysis you cannot distinguish between Z>l+l- and photon>l+l-.

Reply:
That's right, it was a choice to make `madspin` work as well, 2HDMCalculator doesn't provide the BR and partial width for such decay so that I can add them to the` param_card` in order to get the correct total width.
It is also a question of efficiency, knowing that we will not lower the mass cut to capture these photons, we have decided to remove them!

Reply:
you mean 2HDMCalc does not provide BR for "h2 > h3 a"?

Reply:
yes, exactly.